server/app/incident_service.go

```diff
@@ -778,9 +779,54 @@ func (s *IncidentServiceImpl) UpdateStatus(incidentID, userID string, options St
 		}()
 	}
 
+	if options.Status == StatusArchived && incidentToModify.ExportChannelOnArchiveEnabled {
+
+		fileID, err := s.exportFileToChannel(incidentToModify.Name, incidentToModify.OwnerUserID, incidentToModify.ChannelID)
+		if err != nil {
+			_, _ = s.poster.PostMessage(incidentToModify.ChannelID, "Incident Collaboration failed to export channel information. Contact your System Admin for more information.")
+			return errors.Wrap(err, "failed to export file to a channel")
+		}
+
+		if err = s.poster.PostDM(incidentToModify.OwnerUserID, &model.Post{Message: fmt.Sprintf("Incident %s exported succesfully", incidentToModify.Name), FileIds: []string{fileID}}); err != nil {
+			return errors.Wrap(err, "failed to send exported channel result to incident's commander")
```

Comment:
We recently changed the terminology from commander to owner:
```suggestion
			return errors.Wrap(err, "failed to send exported channel result to incident's owner")
```

Reply:
Done

Reply:
:tada: